### PR TITLE
city model created

### PR DESCRIPTION
**Pull request by Nharshimha5**

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+    city = models.CharField(max_length=200)
+    name = models.CharField(max_length=200)
+    state = models.CharField(max_length=200)
+    status=models.CharField(max_length=200)
+    created_date = models.DateTimeField(default=timezone.now)
+    
+
+    def publish(self):
+        self.created_date = timezone.now()
+        self.save()
```
Comment (rajeshrapally) [inline users/models.py:18]: function publish is not required

File: users/models.py [hunk 2]
```diff
@@ -1,3 +1,21 @@
 from django.db import models
 
 # Create your models here.
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+class City(models.Model):
+    city = models.CharField(max_length=200)
```
Comment (rajeshrapally) [inline users/models.py:9]: remove city

File: users/models.py [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+    name = models.CharField(max_length=200)
+    state = models.CharField(max_length=200)
+    status=models.CharField(max_length=200)
```
Comment (rajeshrapally) [inline users/models.py:12]: status should be booleanField with default True

**Review by rajeshrapally — CHANGES_REQUESTED** [review]
Correct that Errors, and migrations files are missing

**Review by rajeshrapally — APPROVED** [review]

**Review by nagpursai — APPROVED** [review]
LGTM

**Pull request merged**